src/features/linodes/LinodesDetail/LinodeResize/LinodeResize.tsx

```
@@ -81,7 +98,8 @@ type CombinedProps = WithTypesProps &
 export class LinodeResize extends React.Component<CombinedProps, State> {
   state: State = {
     selectedId: '',
-    isLoading: false
+    isLoading: false,
+    autoDiskResize: true
```

I think the initial state should take into account whether or not this can actually be enabled.

If I have a Linode with multiple disks, the option is disabled, but we're still sending `allow_auto_disk_resize: true` to the API. Right now the API silently ignores this, but we should update the FE in case the API ever starts returning errors in this situation.